### DISCUSSION TITLE
Tickets at the door?

### DISCUSSION THREAD

**Discussion by roysinn** [liedekef/events-made-easy#139] [Q&A] [answered]
My group has an event that offers registration for advance tickets up to a deadline date. For this particular event, we are offering "tickets at the door" until a set time. I've set up a cut-off time but I would like to present a message on the RSVP form that says something to the effect of "Advance tickets are no longer available. BUT, you can still get tickets at the door...". I see a GLOBAL option to change the "RSVP no longer allowed text" but I don't want to change this text for all events as we have other events that require registration and "tickets at the door" are not an option. Any ideas.

**Comment by liedekef** [comment, marked as answer]
using eme_if and #_IS_RSVP_ENDED you can achieve this.

**Reply by roysinn** [reply]
Great. Many thanks Franky.
